scripts/update-versions.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+if [ "$#" -ne 1 ]; then
+	echo "usage: $(basename $0) <NEW_VERSION>" > /dev/stderr
+	exit 1
+fi
+version=$1
+
+toml_cli=${TOML_CLI:-${CARGO_HOME:-~/.cargo}/bin/toml}
```

Comment:
Would something like [`cargo-release`](https://github.com/crate-ci/cargo-release) work instead?

Reply:
Possibly--I don't know well what packaging tools exist outside of Cargo itself, and the Rust/Cargo docs didn't point to cargo-release. I just wanted to automate what I knew I needed done both to make sure I didn't miss anything and as documentation for the next person to do a release.